manager/controllers/app/fybrik_notebook_write_flow_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"fybrik.io/fybrik/connectors/katalog/pkg/apis/katalog/v1alpha1"
```

Comment:
katalog/v1alpha1 -> katalog/v1beta1?

Reply:
I did not promote Katalog's API, it is our proprietary interface, so I think it's better to leave it in the alpha stage.